### PR TITLE
Fix spacing on register controller form

### LINKED ISSUE #866: Fix spacing on register controller message

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Fix spacing on register controller form

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to /controllers
- Click 'Register new controller'
- Check that spacing of help text next to CTA button displays correctly

## Details

Fixes #866

### CODE REVIEW

File: src/pages/Controllers/_controllers.scss [hunk 1]
```diff
@@ -80,6 +88,8 @@
     }
 
     .p-form-help-text {
+      margin-top: 0;
```
Comment (anthonydillon) [inline src/pages/Controllers/_controllers.scss:91]: This is causing a large gap between inputs and help text. I think [Vanilla default](https://vanillaframework.io/docs/base/forms#stacked) looks better here.

File: src/pages/Controllers/Controllers.js [hunk 2]
```diff
@@ -375,20 +375,18 @@ function RegisterAController({ onClose, showRegisterAController }) {
...
-              disabled={!formValues.certificateAccepted}
-            >
-              Add Controller
-            </button>
-            <p className="p-form-help-text">
-              The credentials are stored locally in your browser and can be
-              cleared on log-out.
-            </p>
-          </div>
+        <div className="register-a-controller__submit-segment">
```
Comment (anthonydillon) [inline src/pages/Controllers/Controllers.js:378]: Why have custom classes here? This should all be achievable with Vanilla. For example:
```html
<div class="row">
  <hr>
</div>
<div class="row u-equal-height">
  <div class="col-8 u-vertically-center">
    <p class="p-form-help-text">The credentials are stored locally in your browser and can be cleared on log-out.</p>
  </div>
  <div class="col-4">
    <button type="submit" disabled="" class="p-button--positive">Add Controller</button>
  </div>
</div>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks better in the footer of the form but the help text is quite far away now. Suggestions inline.

File: src/pages/Controllers/_controllers.scss (hunk 1 above)
Comment (barrymcgee) [reply src/pages/Controllers/_controllers.scss:91]: I agree

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

File: src/pages/Controllers/Controllers.js [hunk 3]
```diff
@@ -375,20 +375,21 @@ function RegisterAController({ onClose, showRegisterAController }) {
             </label>
           </div>
         </div>
-        <div className="row register-a-controller__submit-segment push-1-rem">
```
Comment (anthonydillon) [inline src/pages/Controllers/Controllers.js:378]: Can you remove the `.register-a-controller__submit-segment` styling

Comment (anthonydillon) [reply src/pages/Controllers/Controllers.js:378]: Also the `push-1-rem` styling.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Can you remove the unused styling

**Comment by anthonydillon** [comment]
We now have a double border at the bottom of the form. I suggest removing the `horizontal-rule` class and the styling for it.

**Comment by barrymcgee** [comment]
This change conflicts with my panels branch and this issue will be resolved there.

**Pull request closed without merging**